### PR TITLE
Assignment submission

### PR DESCRIPTION
**Pull request by anandanimesh98**

### CODE REVIEW

**Review by rohit-mp — CHANGES_REQUESTED** [review]
Global variables when used inside functions or parenthesis need to be declared as global. Check out this [link](https://docs.julialang.org/en/v1/manual/variables-and-scoping/index.html#Local-Scope-1)

**Comment by anandanimesh98** [comment]
Changes made

File: JuliaGPU/session1/Animesh/q2.jl [hunk 1]
```diff
@@ -3,10 +3,10 @@ if n>0
   println(n)
   while(n!=1)
     if n%2==0
-       n = n/2
+       global n = n/2
```
Comment (rohit-mp) [inline JuliaGPU/session1/Animesh/q2.jl]: This operation changes ```n``` to ```Float``` but the values are always integers. So could you print out only integers values? (no decimals needed)

**Comment by anandanimesh98** [comment]
Done

**Review by rohit-mp — APPROVED** [review]

**Pull request merged**